[BUG] Template registration without headers and footers results in a validation error in GET API

**Describe the bug**

```
[2021-06-08 06:28:02 +0000] [14447] [ERROR] Exception in ASGI application

Traceback (most recent call last):

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/uvicorn/protocols/http/h11_impl.py", line 394, in run_asgi

    result = await app(self.scope, self.receive, self.send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__

    return await self.app(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/fastapi/applications.py", line 199, in __call__

    await super().__call__(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/applications.py", line 111, in __call__

    await self.middleware_stack(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__

    raise exc from None

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__

    await self.app(scope, receive, _send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/middleware/base.py", line 25, in __call__

    response = await self.dispatch_func(request, self.call_next)

  File "/app/ibet-Prime/app/main.py", line 50, in api_call_handler

    response = await call_next(request)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/middleware/base.py", line 45, in call_next

    task.result()

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/middleware/base.py", line 38, in coro

    await self.app(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__

    raise exc from None

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__

    await self.app(scope, receive, sender)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/routing.py", line 566, in __call__

    await route.handle(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle

    await self.app(scope, receive, send)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/starlette/routing.py", line 41, in app

    response = await func(request)

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/fastapi/routing.py", line 209, in app

    response_data = await serialize_response(

  File "/home/apl/.pyenv/versions/3.8.7/lib/python3.8/site-packages/fastapi/routing.py", line 126, in serialize_response

    raise ValidationError(errors, field.type_)

pydantic.error_wrappers.ValidationError: 2 validation errors for LedgerTemplateResponse

response -> headers

  none is not an allowed value (type=type_error.none.not_allowed)

response -> footers

  none is not an allowed value (type=type_error.none.not_allowed)
```

**Expected behavior**

- Return None

--- a/app/model/schema/ledger.py
+++ b/app/model/schema/ledger.py
@@ -169,23 +169,23 @@ class RetrieveLedgerHistoryResponse(BaseModel):
 class LedgerDetailsDataTemplateResponse(BaseModel):
     """Ledger Details Data Template schema (Response)"""
     type: str
-    source: str
+    source: Optional[str]
 
 
 class LedgerDetailsTemplateResponse(BaseModel):
     """Ledger Details Template schema (Response)"""
     token_detail_type: str
-    headers: List[dict]
+    headers: Optional[List[dict]]
     data: LedgerDetailsDataTemplateResponse
-    footers: List[dict]
+    footers: Optional[List[dict]]
 
 
 class LedgerTemplateResponse(BaseModel):
     """Ledger Template schema (Response)"""
     token_name: str
-    headers: List[dict]
+    headers: Optional[List[dict]]
     details: List[LedgerDetailsTemplateResponse]
-    footers: List[dict]
+    footers: Optional[List[dict]]
 
 
 class LedgerDetailsDataListAllResponse(BaseModel):

--- a/tests/test_app_routers_ledger_{token_address}_template_GET.py
+++ b/tests/test_app_routers_ledger_{token_address}_template_GET.py
@@ -219,6 +219,66 @@ class TestAppRoutersLedgerTokenAddressTemplateGET:
             ],
         }
 
+    # <Normal_2>
+    # All optional items are None
+    def test_normal_2(self, client, db):
+        user = config_eth_account("user1")
+        issuer_address = user["address"]
+        token_address = "0xABCdeF1234567890abcdEf123456789000000000"
+
+        # prepare data
+        _token = Token()
+        _token.type = TokenType.IBET_STRAIGHT_BOND
+        _token.tx_hash = ""
+        _token.issuer_address = issuer_address
+        _token.token_address = token_address
+        _token.abi = {}
+        db.add(_token)
+
+        _template = LedgerTemplate()
+        _template.token_address = token_address
+        _template.issuer_address = issuer_address
+        _template.token_name = "テスト原簿"
+        _template.headers = None  # optional
+        _template.footers = None  # optional
+        db.add(_template)
+
+        _details_1 = LedgerDetailsTemplate()
+        _details_1.token_address = token_address
+        _details_1.token_detail_type = "権利_test_1"
+        _details_1.headers = None  # optional
+        _details_1.footers = None  # optional
+        _details_1.data_type = LedgerDetailsDataType.IBET_FIN
+        _details_1.data_source = None  # optional
+        db.add(_details_1)
+
+        # request target API
+        resp = client.get(
+            self.base_url.format(token_address=token_address),
+            headers={
+                "issuer-address": issuer_address,
+            }
+        )
+
+        # assertion
+        assert resp.status_code == 200
+        assert resp.json() == {
+            "token_name": "テスト原簿",
+            "headers": None,
+            "details": [
+                {
+                    "token_detail_type": "権利_test_1",
+                    "headers": None,
+                    "data": {
+                        "type": "ibetfin",
+                        "source": None,
+                    },
+                    "footers": None,
+                }
+            ],
+            "footers": None,
+        }
+
     ###########################################################################
     # Error Case
     ###########################################################################

